server/chat-commands/info.ts

```diff
@@ -2644,6 +2645,29 @@ export const commands: ChatCommands = {
 	},
 	showhelp: [`/show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`],
 
+	regdate: 'registertime',
+	regtime: 'registertime',
+	async registertime(target, room, user) {
+		this.checkChat();
+		this.runBroadcast();
+		if (!user.autoconfirmed) return this.errorReply(`Only autoconfirmed users can use this command.`);
+		target = toID(target);
+		if (!target) target = user.id;
+		const raw = await Net(`https://${Config.routes.root}/users/${target}.json`).get().catch(e => {
+			if (e.message.includes('Not found')) throw new Chat.ErrorMessage(`User '${target}' is unregistered.`);
+			else throw new Chat.ErrorMessage(e.message);
+		});
```

Comment:
I know you wouldn't be able to use `const` anymore, but I really think you should be using try-catch instead of this approach.

Reply:
Mostly, I don't like mixing approaches like this. I think if you're going to use async/await, you should use all the async/await patterns, like try/catch.

Reply:
```suggestion
		const raw = await Net(`https://${Config.routes.root}/users/${target}.json`).get().catch(e => {
			if (e.message.includes('Not found')) throw new Chat.ErrorMessage(`User '${target}' is unregistered.`);
			else throw new Chat.ErrorMessage(e.message);
		});
```
```suggestion
		let rawResult;
		try {
			rawResult = await Net(`https://${Config.routes.root}/users/${target}.json`).get();
		} catch (e) {
			if (e.message.includes('Not found')) throw new Chat.ErrorMessage(`User '${target}' is unregistered.`);
			throw new Chat.ErrorMessage(e.message);
		}
```